sympy/functions/elementary/exponential.py

```diff
@@ -534,8 +534,13 @@ def _eval_as_leading_term(self, x, logx=None, cdir=0):
         arg0 = arg.subs(x, 0)
         if arg is S.NaN:
             return S.NaN
-        if isinstance(arg0, AccumBounds) and arg0.min == S.NegativeInfinity:
-            return AccumBounds(exp(arg0.min), exp(arg0.max))
+        if isinstance(arg0, AccumBounds):
+            # This check addresses a corner case involving AccumBounds.
+            # if isinstance(arg, AccumBounds) is True, then arg0 can either be 0,
+            # AccumBounds(-oo, 0) or AccumBounds(-oo, oo).
+            # Check out function: test_issue_18473() in test_exponential.py and
+            # test_limits.py for more information.
+            return AccumBounds(0, exp(arg0.max))
```

Comment:
Could this be `return exp(arg0)`?

Reply:
Yeah sure why not , slipped out of my mind , should have caught that !